packages/babel-parser/data/schema.json

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
This schema was generated from `typings/babel-parser.d.ts` by https://github.com/YousefED/typescript-json-schema. I have tuned a little bit: replacing `minItems: 2, maxItems: 2` to `additionalItems: false`.

We should explore generating API option types from JSON schema or vice.